graphbolt/include/graphbolt/cuda_sampling_ops.h

```diff
@@ -58,6 +58,11 @@ c10::intrusive_ptr<sampling::FusedSampledSubgraph> SampleNeighbors(
     bool replace, bool layer, bool return_eids,
     torch::optional<torch::Tensor> type_per_edge = torch::nullopt,
     torch::optional<torch::Tensor> probs_or_mask = torch::nullopt,
+    torch::optional<std::vector<int64_t>> local_node_offsets = torch::nullopt,
```

Comment:
maybe seed_offsets is better, and move this next to `torch::optional<torch::Tensor> nodes`.
We should consider rename all nodes to seeds in the sampling code.